CONTRIBUTING.md

```diff
@@ -7,9 +7,9 @@ Plugin implementations are split across two areas; yaml checks and python
 extensions. The yaml checks (defs) are a way to define analysis in a high-level
 language with the option to leverage python code (core) and the extensions are
 a way to implement functionality like summary output. All output is collected
-and output by default as yaml in a "summary".  
+and output by default as yaml in a "summary".
 
-## Key Components:
+## Key Components
 
 ### defs
 
```

Comment:
I guess since you're fixing things, the README link below is a 404, I think it should be:

hotsos/defs/README.md

And I don't know why github won't let me put the comment below :(

Reply:
Thanks, good catch!